File: CloudFlow/FrontEnd-App/src/app/shared/model/content/search/document-selection-search-result-view.model.ts
import { JsonObject, JsonProperty } from "json2typescript";
import { JsonDateConverter } from "./../../../json-mapper";

@JsonObject
export class DocumentSelectionSearchResultViewModel {
	
	@JsonProperty("documentLocationRealName", String)
	public documentLocationRealName: string = null;

	@JsonProperty("documentId", String)
	public documentId: string = null;

	@JsonProperty("documentName", String)
	public documentName: string = null;
	
	@JsonProperty("author", String)
	public author: string = null;
	
	@JsonProperty("createdDate", JsonDateConverter)
	public createdDate: Date = null;
}

File: CloudFlow/src/ro/cloudSoft/cloudDoc/presentation/client/shared/model/content/search/DocumentSelectionSearchResultViewModel.java
package ro.cloudSoft.cloudDoc.presentation.client.shared.model.content.search;

import java.util.Date;

public class DocumentSelectionSearchResultViewModel {

	private String documentLocationRealName;
	private String documentId;
	private String documentName;
	private String author;
	private Date createdDate;
	
	public String getDocumentLocationRealName() {
		return documentLocationRealName;
	}
	public void setDocumentLocationRealName(String documentLocationRealName) {
		this.documentLocationRealName = documentLocationRealName;
	}
	public String getDocumentId() {
		return documentId;
	}
	public void setDocumentId(String documentId) {
		this.documentId = documentId;
	}	
	public String getDocumentName() {
		return documentName;
	}
	public void setDocumentName(String documentName) {
		this.documentName = documentName;
	}
	public String getAuthor() {
		return author;
	}
	public void setAuthor(String author) {
		this.author = author;
	}
	public Date getCreatedDate() {
		return createdDate;
	}
	public void setCreatedDate(Date createdDate) {
		this.createdDate = createdDate;
	}
}
